src/oneMean/index.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: One Means
+---
+
+<link rel="stylesheet" href="oneMean.css">
+<div id="one-mean">
+  <h2>Step One: Enter Data or Drop CSV</h2>
+  <div id="one-mean-data">
+    <div id="one-mean-data-lhs">
+      <div class="control">
+        <h3>Enter CSV:</h3>
+        <textarea id="csv-input">
+</textarea>
+        <button id="load-data-btn" class="btn">Load Data</button>
+      </div>
+      <select id="variable">
+        <option value="null">Please Select Variable</option>
+      </select>
+    </div>
+    <div class="view">
+      <div class="chart-wrap"><canvas id="data-chart-1"></canvas></div>
+    </div>
+  </div>
+  <h2>Step Two: Run Simulations</h2>
+  <div id="one-mean-sample">
+    <div class="control">
+      <a onclick="twoMean.runSim()" onclick="" class="btn">Run Simulation</a>
```

Comment:
nitpick: Can you change this to a button?

Reply:
ty